executor/executor.go

```diff
@@ -122,7 +123,7 @@ func (e *baseExecutor) Open(goCtx goctx.Context) error {
 	return nil
 }
 
-// Close implements the Executor Close interface.
+// Close closes all executors and release all resource.
```

Comment:
release all the resources.

Reply:
Well, I was thinking "resource" is a uncountable noun.